src/sequences/claw.cpp

```diff
@@ -64,6 +64,11 @@ void Claw::set_arm_position(int8_t side, int8_t position) {
 }
 
 void Claw::fold(bool left_surface) {
+  RCServo0.attach(SERVO_0());
```

Comment:
This doesn't fix the issue. because you will still have delays. between attaching a servo and actually controlling the servo?